pkg/version/version.go

```diff
@@ -26,7 +27,7 @@ func GetVersionInfo() map[string]string {
 	versionInfo["DDEV version"] = versionconstants.DdevVersion
 	versionInfo["ddev-environment"] = environment.GetDDEVEnvironment()
 	versionInfo["cgo_enabled"] = strconv.FormatInt(versionconstants.CGOEnabled, 10)
-	versionInfo["global-ddev-dir"] = globalconfig.GetGlobalDdevDir()
+	versionInfo["global-ddev-dir"] = util.WindowsPathToCygwinPath(globalconfig.GetGlobalDdevDir())
```

Comment:
I guess this is the other place.  Could (or should)? GetGlobalDdevDir() have the responsibility for the transformation?

Reply:
I will look into this after this PR goes in.

I don't want to make any Windows related changes here, because it's a fork.